src/Compilers/VisualBasic/Test/Symbol/SymbolsTests/Source/OverridesTests.vb

```diff
@@ -7213,5 +7213,63 @@ End Class
 void ValidatorBase<T>.DoValidate(T objectToValidate)")
         End Sub
 
+        <Fact>
+        Public Sub OverrideOfCSharpCovariantReturn()
```

Comment:
>OverrideOfCSharpCovariantReturn [](start = 19, length = 31)

I think we should also add a test for a success scenario where VB overrides  with return type String and calling through Base and Derived calls the overrides.

Reply:
I will add the compile-time success test. Runtime tests will have to wait. I will show the IL at the call site.

---
In reply to: [424732466](https://github.com/dotnet/roslyn/pull/44025#discussion_r424732466) [](ancestors = 424732466)